core/tests/org.fusesource.ide.camel.tests.util/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,10 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: JBoss Fuse Tooling Tests Util
+Bundle-SymbolicName: org.fusesource.ide.camel.tests.util
+Bundle-Version: 8.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Require-Bundle: org.junit;bundle-version="4.12.0",
+ org.fusesource.ide.camel.model.service.core;bundle-version="8.0.0",
+ org.fusesource.ide.camel.model.service.v2_16_1;bundle-version="8.0.0"
```

Comment:
We should try to not use / rely on the service implementation bundles. Service-core should be all we work with.